packages/headless/src/features/result-preview/__snapshots__/result-preview-insight-analytics-actions.test.ts.snap

```diff
@@ -0,0 +1,24 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`#logDocumentQuickview when analyticsMode is \`next\` should call relay.emit properly 1`] = `
+[
+  "InsightPanel.ItemAction",
+  {
+    "action": "preview",
```

Comment:
Should we also send the `sourceEntityId`?

https://github.com/coveo-platform/analytics_schema/blob/master/schemas/client/api.event.insightPanel.itemAction/1/api.event.insightPanel.itemAction.1.request.json#L41

Reply:
good question, since it exists in the schema I think we should yeah, however I see that we are not sending in all the insight panel events, so I will create a jira to investigate and updated this, we basically need to figure out what to send as the value of this key and add it to all the events where it should be sent.